### PR TITLE
Fix: Package Electron in production

### LINKED ISSUE #62: Electron packaging tools aren't installed with production builds
If you're creating a production build as part of a release process, you run `npm install --production` to skip all the development dependencies. The gotcha is that the Electron packaging tools get skipped as well. Those need to [move out of the `devDependencies` section](https://github.com/rapid7/awsaml/blob/v1.2.0/package.json#L44-L45) of the package.json file and into the `dependencies` section. Both the `electron-packager` and `electron-prebuilt` modules should move.

There's already [an `--ignore=node_modules/electron` flag](https://github.com/rapid7/awsaml/blob/v1.2.0/package.json#L19) in the build process, so we don't have to worry about those packages making it into the released builds.

### PR DESCRIPTION
**Pull request by fmitchell-r7**
This fixes #62. You can now run `npm install --production` and be able to build a working copy of Awsaml with the installed modules.

In addition to the Electron build tool chain, I made the `babel-preset-*` family of modules (introduced in #53) part of the production install as well. I excluded tests from production builds and updated the README to reflect the new build process.

### CODE REVIEW

**Comment by rvanburen-r7** [comment]
LGTM 👍

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "electron-start": "electron .",
     "local-start": "node local.js",
     "prebuild": "rm -rf dist/",
-    "build": "electron-packager . Awsaml --asar --ignore=node_modules/electron --ignore=dist --out=dist --platform=darwin,linux,win32 --arch=x64 --version=0.37.8 --app-version=${npm_package_version} --app-bundle-id=com.rapid7.awsaml --helper-bundle-id=com.rapid7.awsaml.helper",
+    "build": "electron-packager . Awsaml --asar --ignore=node_modules/electron --ignore=test --ignore=dist --out=dist --platform=darwin,linux,win32 --arch=x64 --version=0.37.8 --app-version=${npm_package_version} --app-bundle-id=com.rapid7.awsaml --helper-bundle-id=com.rapid7.awsaml.helper",
```
Comment (erran-r7) [inline package.json]: @fmitchell-r7 what's the benefit of ignoring `dist` if we remove it in the `prebuild` script?

Comment (erran-r7) [inline package.json]: @fmitchell-r7 Ignore me, I'm reading this diff wrong. 😄 I realise now you actually added `--ignore=test`.

**Comment by erran-r7** [comment]
:shipit: Looks good from here too!

**Pull request merged**